### PR TITLE
Used webbroswer module in docs Makefile.

### PR DESCRIPTION
**Pull request by pauloxnet**
#### Trac ticket number

"N/A"

#### Branch description

Used webbroswer module in docs Makefile.

#### Checklist
- [x] This PR targets the `main` branch.
- [x] The commit message is written in past tense, mentions the ticket number, and ends with a period.
- [ ] I have checked the "Has patch" ticket flag in the Trac system.
- [ ] I have added or updated relevant tests.
- [ ] I have added or updated relevant docs, including release notes if applicable.
- [ ] I have attached screenshots in both light and dark modes for any UI changes.

### CODE REVIEW

File: docs/Makefile [hunk 1]
```diff
@@ -61,7 +61,7 @@ html:
 	@echo "Build finished. The HTML pages are in $(BUILDDIR)/html."
 
 htmlview: html
-	$(PYTHON) -c "import webbrowser; webbrowser.open('_build/html/index.html')"
+	$(PYTHON) -m webbrowser $(BUILDDIR)/html/index.html
```
Comment (felixxm) [inline docs/Makefile:64]: I would add quotations.
```suggestion
	$(PYTHON) -m webbrowser "$(BUILDDIR)/html/index.html"
```

**Review by felixxm — APPROVED** [review]
Makes sense, thanks :+1:

File: docs/Makefile (hunk 1 above)
Comment (pauloxnet) [reply docs/Makefile:64]: I tried to be aligned with other commands in the Makefile which don't have quotations.
Maybe we can review the file in another PR and add quotations?

Comment (felixxm) [reply docs/Makefile:64]: I don't think that quotations are needed everywhere. I'd add them here to follow an example from Python docs: https://docs.python.org/3/library/webbrowser.html#module-webbrowser.

Comment (pauloxnet) [reply docs/Makefile]: Ok, make sense. I updated the commit.

**Review by sarahboyce — APPROVED** [review]
Thank you @pauloxnet :star:

**Pull request merged**